project/src/main/java/com/google/sps/servlets/DeleteProductServlet.java

```diff
@@ -56,7 +56,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     datastore.delete(entity.getKey()); // Delete from the product table
 
     //Delete from product search Api
-    ProductSearchLibrary.deleteProduct(productId);
+    //ProductSearchLibrary.deleteProduct(productId);
```

Comment:
why are you removing this?

Reply:
Neel wanted me to remove it. I think it's for Phillips code, but idk for sure. I'm guessing once we get it working we'll switch everything back